content/blog_posts/articles/hmr-in-adonisjs.md

```diff
@@ -0,0 +1,106 @@
+---
+summary: Hot Module Replacement (HMR) is now available in AdonisJS.
+---
+
+We're thrilled to announce that HMR is now available in AdonisJS!
+
+In short: HMR now allows us to modify the code of our controllers and their dependencies without having to restart the entire dev server each time. This results in a much faster feedback loop and a significantly improved development experience.
+
+## Hot Hook
+
+We've developed [Hot-Hook](https://github.com/Julien-R44/hot-hook), a framework-agnostic library that enables HMR in Node.JS with ESM. The library works with any framework. It's also important to note that Hot-hook **does not perform any static analysis, heavy code transformations, AST parsing, or bundling**.
+
+The approach is very simple but works perfectly. 
```

Comment:
```suggestion

```

Reply:
why remove this ?

Reply:
What is the purpose of the sentence?

Reply:
saying that the approach is super simple compared to alternative solutions that use either bundling, static analysis or lot of code transformation

Reply:
Because this sentence is not needed in the article and it does not follow the tone of the article.

Reply:
When I read it, it feels like you say simple things are bad by default